partials/content.php

```diff
@@ -10,19 +10,19 @@
 ?>
 
 <?php tha_entry_before(); ?>
-<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>
+<article class="entry" id="post-<?php the_ID(); ?>" <?php post_class(); ?>
```

Comment:
can you remove this class and open an issue to add this since this will add 2 class elements to the div.